src/om_bootstrap/progress_bar.cljs

```diff
@@ -3,61 +3,65 @@
   work."
   (:require [om-bootstrap.types :as t]
             [om-tools.dom :as d :include-macros true]
-            [schema.core :as s])
-  (:require-macros [schema.macros :as sm]))
+            [schema.core :as s :include-macros true]
```

Comment:
nope, this doesn't work with the version of schema in this library. We've gotta keep it the old way until we force users to upgrade to Schema 0.3.x.